packages/cli/test/unit/commands/pg/backups/info.unit.test.ts

```diff
@@ -0,0 +1,120 @@
+import {stdout} from 'stdout-stderr'
+import Cmd from '../../../../../src/commands/pg/backups/info'
+import runCommand from '../../../../helpers/runCommand'
+import * as nock from 'nock'
+import expectOutput from '../../../../helpers/utils/expectOutput'
+import {expect} from 'chai'
+import heredoc from 'tsheredoc'
+import stripAnsi = require('strip-ansi')
+
+const shouldInfo = function (cmdRun: (args: string[]) => Promise<any>) {
```

Comment:
This makes it nearly impossible to isolate a single test if ever needed and I don't think this is needed anyway. Does it make sense to remove this wrapper and just call the tests as per usual?

Reply:
I struggled with deciding what to do here. At one point I did remove the wrapper.. but I added it back because of how this works: https://github.com/heroku/cli/blob/main/packages/pg-v5/test/unit/commands/backups/helpers.js .. we're still not 100% sure what we want to do with that. I think there's a story to refactor that, or just remove it. And maybe once we decide on and act on that we can also decide to refactor this. Until then I'd rather leave it as is.